thoth/common/openshift.py

```diff
@@ -1081,6 +1081,36 @@ def schedule_package_extract(
             },
         )
 
+    def schedule_sync_job(
+        self,
+        document_type: Optional[str],
+        force_sync: bool = False,
+        graceful: bool = True,
+        job_id: Optional[str] = None,
+    ) -> Optional[str]:
+        """Schedule graph sync for specific document id."""
+        if not self.backend_namespace:
```

Comment:
We use backend resources to serve user requests. This job might be more suitable for middletier (that was dedicated for data aggregation and ingestion):

```suggestion
        if not self.middletier_namespace:
```